CMakeLists.txt

```diff
@@ -211,6 +211,10 @@ endif(DEFINED WIN32)
 
 add_executable(edge src/edge.c)
 target_link_libraries(edge n2n)
+if(DEFINED WIN32)
+  # Prompt UAC
+  set_target_properties(edge PROPERTIES LINK_FLAGS "/MANIFESTUAC:\"level='requireAdministrator' uiAccess='false'\"")
+endif(DEFINED WIN32)
```

Comment:
The cmake build system is deprecated and will be removed soon.
Can you make an equivalent change to the Makefiles?

Reply:
I see #1042, I haven't tried mingw to build, visual studio complied the cmake version automatically for me

Might try it later